plugins/agent-utilities/skills/fusion-parametric-design/tests/test_pipeline_seams.py

```
@@ -386,6 +386,112 @@ def createInput(self, mesh_body):
+        self.assertAlmostEqual(5.0, fit["parameters"]["radius"], places=6)
+        self.assertAlmostEqual(0.5, support["axis_evidence"]["eigengap"], places=6)
+        # And the boundary the grouping left behind agrees with it, independently.
+        self.assertAlmostEqual(5.0, support["boundary_circle"]["loop_radius"], places=6)
+        self.assertIsNone(support["boundary_circle"]["flag"])
+
+    def test_the_record_the_new_path_produces_still_parses_for_the_planner(self) -> None:
+        """The uncertainty names U3 licenses from are the ones the joint system writes."""
+        dump = self._run_both()
+        record = seg.fit_regions(dump, ts.spec())
+        parsed = parse_fit_record(record)
+        cylinder = [r for r in parsed.regions if r.accepted and r.fit.kind == "cylinder"][0]
```

Ruff (0.16.1)

[warning] Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)